packages/web-components/fast-foundation/src/number-field/number-field.ts

```diff
@@ -261,6 +261,24 @@ export class NumberField extends FormAssociatedNumberField {
     public handleChange(): void {
         this.$emit("change");
     }
+
+    /**
+     * Handles the internal control's `keydown` event
+     * @internal
+     */
+    public handleKeyDown(event: KeyboardEvent): boolean {
+        if (event.key === "ArrowUp") {
```

Comment:
Actually, if you could import the key identifier strings from the shared resource like we do in radio group that would match the new pattern. 
https://github.com/microsoft/fast/blob/79f62ae08fd5db598272ca907ba03c4afa81a49f/packages/web-components/fast-foundation/src/radio-group/radio-group.ts#L2

Reply:
Updated in b28918d.

Reply:
An issue to track alignment across the components: https://github.com/microsoft/fast/issues/5089 :)